backend/target_zone_estimation.py

```diff
@@ -0,0 +1,47 @@
+import json
+import asyncio
+from Step_Time_Calculation import calculate_step_time
+
+
+real_force_data = [
+    (0.00000, 159.548187),  
+    (0.00093, 159.304047),  
+    (0.01296, 158.327484),  
+    (0.01620, 158.327484), 
+    (0.05370, 157.106781),
+    (0.05972, 156.862640)   
+]
+
+threshold = 20.0
+
+async def handle_data_streaming(websocket):
```

Comment:
This function is not called throughout the app so the data that's gotten and calculations done don't make it to the frontend